dir.props

```diff
@@ -520,9 +527,9 @@
 
   <!-- Default Test platform to deploy the netstandard compiled tests to -->
   <PropertyGroup>
-    <TestTFM Condition="'$(TestTFM)'==''">netcoreapp1.0</TestTFM>
-    <!-- we default FilterToTestTFM to netcoreapp1.0 if it is not explicity defined -->
-    <FilterToTestTFM Condition="'$(FilterToTestTFM)'==''">netcoreapp1.0</FilterToTestTFM>        
+    <TestTFM Condition="'$(TestTFM)'==''">netcoreapp1.1</TestTFM>
```

Comment:
Do these tests even work for netcoreapp1.1 yet? Right now I'm not sure I want to move our default to netcoreapp1.1 for the entire repo yet because it isn't clear if that is the TFM we will use for our 1.1 release (despite the numbers matching :()